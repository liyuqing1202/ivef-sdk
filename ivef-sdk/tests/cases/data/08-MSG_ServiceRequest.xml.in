<MSG_IVEF xmlns="urn:http://www.ivef.org/XMLSchema/IVEF/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:http://www.ivef.org/XMLSchema/IVEF/1.1
\\projsrvr\isis\develop\ivef-sdk\trunk\doc\ivef1.xsd">
	<Header MsgRefId="123456789012345678901234567890123456789" Version="String"/>
	<Body>
		<ServiceRequest>
			<Area>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
			</Area>
			<Area>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
			</Area>
			<Area>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
				<Pos Lat="3.1415926535897932384626433832795" Long="3.1415926535897932384626433832795"/>
			</Area>
			<Transmission Type="1" Period="3.1415926535897932384626433832795"/>
			<Item FieldSelector="all" DataSelector="1"/>
			<Filter Predicate="Text"/>
		</ServiceRequest>
	</Body>
</MSG_IVEF>

