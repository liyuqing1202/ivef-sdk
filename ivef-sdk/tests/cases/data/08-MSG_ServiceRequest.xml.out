TestApp ready for input
MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = String
    Body
        ServiceRequest
            Area
                Pos
                    Lat = 3.14159
                    Long = 3.14159
                Pos
                    Lat = 3.14159
                    Long = 3.14159
                Pos
                    Lat = 3.14159
                    Long = 3.14159
            Area
                Pos
                    Lat = 3.14159
                    Long = 3.14159
                Pos
                    Lat = 3.14159
                    Long = 3.14159
                Pos
                    Lat = 3.14159
                    Long = 3.14159
            Area
                Pos
                    Lat = 3.14159
                    Long = 3.14159
                Pos
                    Lat = 3.14159
                    Long = 3.14159
                Pos
                    Lat = 3.14159
                    Long = 3.14159
            Transmission
                Type = 1
                Period = 3.141593
            Item
                DataSelector = 1
                FieldSelector = all
            Filter
                Predicate = Text
<MSG_IVEF xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.iala-to-be-confirmed.org/XMLSchema/IVEF/0.2.5">
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="String"/>
<Body>
<ServiceRequest>
<Area>
<Pos Lat="3.14159" Long="3.14159"/>
<Pos Lat="3.14159" Long="3.14159"/>
<Pos Lat="3.14159" Long="3.14159"/>
</Area>
<Area>
<Pos Lat="3.14159" Long="3.14159"/>
<Pos Lat="3.14159" Long="3.14159"/>
<Pos Lat="3.14159" Long="3.14159"/>
</Area>
<Area>
<Pos Lat="3.14159" Long="3.14159"/>
<Pos Lat="3.14159" Long="3.14159"/>
<Pos Lat="3.14159" Long="3.14159"/>
</Area>
<Transmission Type="1" Period="3.141593"/>
<Item DataSelector="1" FieldSelector="all"/>
<Filter Predicate="Text"/>
</ServiceRequest>
</Body>
</MSG_IVEF>

TestApp shutting down
